ex22_ch4.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+#step1: get inputs from user for three numbers
+number1 = int(input("Enter one number: "))
+number2 = int(input("Enter a second number: "))
+number3 = int(input("Enter a third number: "))
+
+#step2: set up inequalities to make sure all numbers are different. Then, set up another conditional if the numbers are different.
+if number1 == number2 or number1 == number3 or number2 == number3:
+    print("The numbers are not different. Game over.")
+elif number1 != number2 or number1 != number3 or number2 != number3:
```

Comment:
I think you can use an `else` here and not have to do the conditional. If the numbers are different (which you test for on line 14), then you know they are different and can just proceed with the else.